client/ionic/src/pages/ProtectYourself.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { FlowLoader, Flow, LoadedFlow } from '../content/flow';
+import {
+  IonContent,
+  IonPage,
+  IonSlides,
+  IonSlide,
+  IonCard,
+  IonCardContent,
+  IonImg,
+} from '@ionic/react';
+import TopNav from '../components/TopNav';
+import 'tachyons';
+import { getUserContext } from '../content/userContext';
+
+function useDynamicFlow(id: string) {
```

Comment:
make a TODO to me for DRY, sharing this code

Reply:
I'll also work on that right now

Reply:
done